test/integration/client_test.cpp

```diff
@@ -150,14 +161,17 @@ TEST_F(ClientServerTest, SendTxWhenInvalidJson) {
       "\"2323232323232323232323232323232323232323232323232323232323232323\"\n"
       "  }]\n"
       "}";
```

Comment:
learn how [raw strings](https://solarianprogrammer.com/2011/10/16/cpp-11-raw-strings-literals-tutorial/) in C++ work. This can be significantly simplified.